internal/environments/environments_test.go

```diff
@@ -0,0 +1,99 @@
+// Tests for the EnvironmentController
+
+package environments
+
+import (
+	"log"
+	"testing"
+
+	//"github.com/broadinstitute/sherlock/internal/sherlock"
+	"github.com/broadinstitute/sherlock/internal/db"
+	"github.com/broadinstitute/sherlock/internal/tools"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/assert"
+	"gorm.io/gorm"
+)
+
+func TestCreateEnvironments(t *testing.T) {
```

Comment:
since this is by our definition an integration Test (it uses a real postgres instance). Can you add `Integration` somewhere in the name of the test function. The make tooling looks for this to selectively run just integration tests.